modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -279,6 +279,61 @@ autoSaveAsDraftTest(
 	}
 );
 
+baseTest(
+	'LPD-31427: select web content display template with the Preview feature',
+	async ({journalEditArticlePage, page, site}) => {
+		page.on('dialog', (dialog) => dialog.accept());
+
+		await journalEditArticlePage.goto({siteUrl: site.friendlyUrlPath});
+
+		const title = getRandomString();
+
+		await page.getByText('Content', {exact: true}).waitFor();
+
+		await journalEditArticlePage.titleInput.fill(title);
+
+		await page.getByRole('button', {name: 'Publish'}).click();
+
+		await journalEditArticlePage.editArticle(title);
+
+		await page.getByText('Content', {exact: true}).waitFor();
+
+		await page.getByRole('button', {name: 'Clear'}).click();
```

Comment:
which `clear` button is this one? Is the click really needed?

Reply:
![image](https://github.com/user-attachments/assets/fc444a7d-dd1d-420d-bea8-576e51583838)
This clear button under the templates.

Reply:
I need to clear the selection to make sure the selecting a template is possible trough the preview feature.

Reply:
I did not know that the browser could access the cached menu states. I added steps to the test to open the template so the locator could find the correct button.